### ISSUE TITLE
Builds for 3.3 don't include `configuration-notices`. 

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt-distribution#28]

**Comment by bobdenotter** [comment]
.. which is odd, since i would think this would take care of that: 

```
    if (( $(echo "$MAJOR_MINOR_VER > 3.2" | bc -l) )); then
        $_PACKAGES="$_PACKAGES bolt/configuration-notices:^1.0@dev"
    fi
```

@GawainLynch Can you look into this?

**Comment by GawainLynch** [comment]
Oh … well … what ~~idiot~~ lovely person wrote that?

The fist character on the second line is _roughly_ the equivalence to `$$_PACKAGES` in PHP :disappointed:

**Comment by GawainLynch** [comment]
Fixed in 2c9ed2d3ead03f7c1b9bc0be8f3dcda0e9d0f725
